.github/workflows/haskell-ci.yml

```
@@ -21,25 +25,99 @@ jobs:
     strategy:
       matrix:
         ghc-version: [latest, 9.12, "9.10", 9.8, 9.6]
```

```suggestion
        ghc-version: ["9.12", "9.10", "9.8", "9.6"]
```

Currently, `latest` and `9.12` are the same, so this runs the 9.12 build twice. Also, we usually prefer not to be surprised by a sudden version bump in what `latest` refers to.